### PR TITLE
added Modal to Portfolio Page

### PR DESCRIPTION
**Pull request by denysM789**
- general Modal added to Portfolio Page
- user-data not yet saved in Redux store
- CSS not yet at 100%

### CODE REVIEW

File: src/components/Portfolio/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  const { width: screenWidth, height: screenHeight } = useWindowSize();
+
+  const [showCalendar, setShowCalendar] = useState(false);
+  const [selectedDate, setSelectedDate] = useState(null);
+  const [showInput, setShowInput] = useState(false);
+
+  const openCalendar = () => {
+    setShowCalendar(!showCalendar);
+  };
+  useEffect(() => {}, [showCalendar]);
```
Comment (yoloOnTheBattlefield) [inline src/components/Portfolio/Modal/Modal.js]: no need for this

File: src/components/Portfolio/Modal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+  const openInput = () => {
+    setShowInput(!showInput);
+  };
+  useEffect(() => {}, [showInput]);
```
Comment (yoloOnTheBattlefield) [inline src/components/Portfolio/Modal/Modal.js]: no need for this

File: src/pages/Portfolio/Portfolio.js [hunk 3]
```diff
@@ -1,7 +1,98 @@
...
+
+  // useEffect(() => {
+  //   const handleClickOutside = () => {
+  //     setShowModal(false);
+  //   };
+  //   document.addEventListener("mousedown", handleClickOutside);
+  //   return () => {
+  //     document.removeEventListener("mousedown", handleClickOutside);
+  //   };
+  // }, [showModal]);
```
Comment (yoloOnTheBattlefield) [inline src/pages/Portfolio/Portfolio.js]: no need for this

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/Portfolio/Modal/Modal.js (hunk 1 above)
Comment (denysM789) [reply src/components/Portfolio/Modal/Modal.js]: deleted the useEffect

File: src/pages/Portfolio/Portfolio.js (hunk 3 above)
Comment (denysM789) [reply src/pages/Portfolio/Portfolio.js:67]: deleted the useEffect

Comment (denysM789) [reply src/pages/Portfolio/Portfolio.js]: done

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**